src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -168,6 +168,12 @@ public override string ToString()
 
         internal bool WasRedirected() => (_sendStatus & MessageIsRedirect) != 0;
 
+        private bool Disposed
+        {
+            get => (_sendStatus & MessageDisposed) != 0;
+            set => _sendStatus |= MessageDisposed;
```

Comment:
This only works for `Disposed = true;`; it's buggy for `Disposed = false;`.  Even if we don't do the latter, if we're going to implement this as a setter, we should make it correct in order to avoid future bugs.